src/universal/.devcontainer/devcontainer.json

```
@@ -9,10 +9,10 @@
             "userUid": "1000",
             "userGid": "1000"
         },
-        "ghcr.io/devcontainers/features/dotnet:1": {
-            "version": "7.0.306",
-            "installUsingApt": "false",
-            "additionalVersions": "6"
+        "ghcr.io/devcontainers/features/dotnet:2": {
+            "version": "8.0.x",
+            "dotnetRuntimeVersions": "7.0.x, 6.0.x",
```

```suggestion
            "dotnetRuntimeVersions": "6.0",
```

For the universal image, we only add support for two of the latest LTS versions. Hence, can we update this?

---

One of the reasons here @samruddhikhandale is that the likely inclusion of folks adding the VS Code extensions for .NET (C#/C# Dev Kit) require .NET 7 currently.  Work is underway to switch to .NET8, but until then it is an added convenience to have the runtimes in this image to avoid an additional acquisition that would slow the experience for using the devcontainer for .NET a bit.  .NET 7 is still fully supported right now.

---

Thanks for the details, appreciate it. 

The `universal` image shouldn't support three versions of a particular tool/framework, at the most it should only support two LTS versions.

> .NET 7 is still fully supported right now.

I agree, however, .NET 7 is STS and not LTS. For `universal` images we aim at supporting only LTS versions.

Given the above reasoning, for a while we can ignore the rule of having only LTS and support .Net 7, but we want to avoid adding a third version (the image is already huge in size, would like to avoid expanding it)

@timheuer would adding .Net 8 and .Net 7 to the image sound good to you?

---

I'd feel good about that decision.

---

```suggestion
            "dotnetRuntimeVersions": "7.0",
```